cmd/clusterctl/validation/validate_pods.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"golang.org/x/net/context"
```

Comment:
why not just import "context"?

Reply:
Fixed here:
https://github.com/kubernetes-sigs/cluster-api/pull/679/files#diff-53767fe464d92e82c3034f3769d689dfR20

https://github.com/kubernetes-sigs/cluster-api/pull/679/files#diff-6334f0ae8a2b2bebd8b635352f2a9426R20